src/dash-table/components/ControlledTable/fragments/TableTooltip.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { PureComponent } from 'react';
+
+import Tooltip, { ITooltipProps, Arrow } from 'dash-table/components/Tooltip';
+import tooltipHelper from 'dash-table/components/tooltipHelper';
+import ReactDOM from 'react-dom';
+import { isEqual } from 'core/comparer';
+
+interface IState {
+    arrow?: Arrow;
+    cell?: any;
+}
+
+export default class TableTooltip extends PureComponent<ITooltipProps, IState> {
+    constructor(props: ITooltipProps) {
+        super(props);
+
+        this.state = {
+            arrow: Arrow.Bottom
+        };
+    }
+
+    updateBounds = (cell: any) => {
+        this.setState({ cell });
+    }
+
+    shouldComponentUpdate(nextProps: ITooltipProps, nextState: IState) {
+        this.adjustPosition();
+        return !isEqual(this.props, nextProps) || !isEqual(this.state, nextState);
+    }
+
+    componentDidUpdate() {
+        this.adjustPosition();
+    }
+
+    render() {
+        const { arrow } = this.state;
+
+        return (<Tooltip
+            key='tooltip'
+            ref='tooltip'
+            arrow={arrow}
+            {...this.props}
+        />);
+    }
+
+    private adjustPosition() {
+        const { cell } = this.state;
+
+        const el = ReactDOM.findDOMNode(this.refs.tooltip) as any;
+
+        const { positionalParent, parent } = tooltipHelper(el, cell);
+
+        if (!positionalParent || !parent || !el) {
+            return;
+        }
+
+        const positionalBounds = positionalParent.getBoundingClientRect();
+        const parentBounds = parent.getBoundingClientRect();
+
+        const { clientWidth: elWidth, clientHeight: elHeight } = el;
+        const elAnchorHeight = Math.max(
+            parseFloat(getComputedStyle(el, ':before').borderWidth || '0'),
+            parseFloat(getComputedStyle(el, ':after').borderWidth || '0')
```

Comment:
The height of the element is not enough.. need to know how much gets added by the arrow -- luckily while we can't style the pseudo-elements from JS, we can still access their existing style.